site/_data/authorsData.json

```
@@ -334,8 +334,11 @@
     "descriptions": {
       "en": "Developer advocate working on Chrome DevTools at Google.",
       "es": "Developer advocate, trabajando en Chrome DevTools en Google.",
-      "zh": "现为谷歌 Chrome DevTools 的开发者推广工程师。",
-      "ru": "Developer advocate, rabotayushchiy nad Chrome DevTools v Google."
+      "zh": "现为谷歌 Chrome DevTools 的开发技术推广工程师。",
+      "ru": "Developer advocate, rabotayushchiy nad Chrome DevTools v Google.",
+      "pt": "Defensor do desenvolvedor trabalhando no Chrome DevTools no Google.",
```

@andreban is this translation correct? (I Google translate it)

---

I wouldn't translate "Developer Advocate", and just use the english term in this case.

```suggestion
      "pt": "Developer Advocate trabalhando no Chrome DevTools no Google.",
```